source/fresh-installation.rst

```diff
@@ -344,8 +344,31 @@ Installing on a Personal Computer
 ---------------------------------
 
 You can install these packages on your personal laptop outside the controls
-network. Install miniconda or Ananconda, and create user environment as
+network. Install miniconda or Anaconda, and create user environments as
 described above. All of the packages are mirrored on anaconda.org, outside of
 the NSLS-II firewall, where you will be able to access them. The channels are
 called ``lightsource2-tag`` and ``lightsource2-dev`` instead of ``nsls2-tag``
-and ``nsls2-dev`` respectively.
+and ``nsls2-dev`` respectively. The following serves as a step by step guide:
+
+1.  Follow the instructions in the link below to install minconda or anaconda.
+        https://conda.io/docs/user-guide/install/index.html
+
+2.  Open a terminal and run the following commands to install the enviroments. 
+
+.. code-block:: bash
+
+    sudo conda create -p /opt/conda_envs/collection-2018-2.1 -c lightsource2-tag -y collection
```

Comment:
I think we should probably not auto-confirm (`-y`) since these commands are being run interactively.

Reply:
For a personal install, use `-n <env-name>`, not `-p <system_path>`, and do it as user, not root. We should probably not auto-confirm (`-y`) since these commands are being run interactively.